### PR TITLE
Update readme & Fix Android Icon render 

### PR DESCRIPTION
**Pull request by girish54321**
1.) Project Structure added in readme
2.) Fixed Android Icon render issues

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,6 +45,70 @@ git clone git@github.com:WrathChaos/react-native-typescript-boilerplate.git my-a
 - **Built-in Custom Font Implementation**
   - All you need to do is copy-paste the .tff file and run `npx react-native-asset` command
 
+## Project Structure
```
Comment (kuraydev) [inline README.md:48]: Awesome idea!

File: README.md [hunk 2]
```diff
@@ -45,6 +45,70 @@ git clone git@github.com:WrathChaos/react-native-typescript-boilerplate.git my-a
...
+├───README.md
+├───app.json
+├───index.js
+├───metro.config.js
+├───package-lock.json
+├───package.json
+├───react-native.config.js
+├───tsconfig.json
+├───yarn-error.log
+└───yarn.lock
```
Comment (kuraydev) [inline README.md]: we do not need these. These are just generated files.

File: package.json [hunk 3]
```diff
@@ -8,7 +8,16 @@
     "start": "react-native start",
     "test": "jest",
     "lint": "eslint . --ext .js,.jsx,.ts,.tsx",
-    "clean-up": "rm -rf .git README.md && rm -rf ./assets/RN-Typescript-Boilerplate.gif ./assets/logo.png && git init"
+    "clean-up": "rm -rf .git README.md && rm -rf ./assets/RN-Typescript-Boilerplate.gif ./assets/logo.png && git init",
+    "postinstall": "jetify",
+    "ios11": "react-native run-ios --simulator=\"iPhone 11\"",
+    "ios8": "react-native run-ios --simulator=\"iPhone 8\"",
+    "ios8p": "react-native run-ios --simulator=\"iPhone 8 Plus\"",
+    "ios11promax": "react-native run-ios --simulator=\"iPhone 11 Pro Max\"",
```
Comment (kuraydev) [inline package.json]: We do not need these, there are too specific for the dev to dev. Devs can add these as optional for themselves.

File: package.json [hunk 4]
```diff
@@ -8,7 +8,16 @@
     "start": "react-native start",
     "test": "jest",
     "lint": "eslint . --ext .js,.jsx,.ts,.tsx",
-    "clean-up": "rm -rf .git README.md && rm -rf ./assets/RN-Typescript-Boilerplate.gif ./assets/logo.png && git init"
+    "clean-up": "rm -rf .git README.md && rm -rf ./assets/RN-Typescript-Boilerplate.gif ./assets/logo.png && git init",
+    "postinstall": "jetify",
```
Comment (kuraydev) [inline package.json]: I think this should be better like: 
`"post-install": "npx jetify"`

File: android/settings.gradle [hunk 5]
```diff
@@ -1,5 +1,3 @@
 rootProject.name = 'RNTypescriptBoilerplate'
-include ':react-native-vector-icons'
-project(':react-native-vector-icons').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-vector-icons/android')
```
Comment (kuraydev) [inline android/settings.gradle:3]: Don't we need these?

**Review by kuraydev — CHANGES_REQUESTED** [review]

**Comment by kuraydev** [comment]
@girish54321 Lovely work man! Thank you so much

File: README.md (hunk 2 above)
Comment (girish54321) [reply README.md]: Ok.

File: package.json (hunk 3 above)
Comment (girish54321) [reply package.json]: Will remove this.

File: package.json (hunk 4 above)
Comment (girish54321) [reply package.json]: Hmm npx right will update this.

File: android/settings.gradle (hunk 5 above)
Comment (girish54321) [reply android/settings.gradle:3]: Nop new package will copy icons for node_modules.

**Comment by girish54321** [comment]
Hi made few changes you can check and let me know.

**Review by kuraydev — APPROVED** [review]

**Comment by kuraydev** [comment]
Thank you so much for the contribution @girish54321 :)

**Pull request merged**